### DISCUSSION TITLE
How to use Renovate to update testing databases (via Docker)?

### DISCUSSION THREAD

**Discussion by janpio** [renovatebot/renovate#14473] [Request Help]
We are using Renovate on a repo that is testing a bunch of databases - MySQL 5.6, 5.7 and 8.x for example. 
How can we make Renovate suggest only useful updates, basically for patch versions?

Example: https://github.com/prisma/prisma-engines/pull/2611/files First one does not make sense, second potentially might.

**Comment by rarkins** [comment]
Example:

```
"separateMinorPatch": true,
"packageRules": [{
  "matchFiles": ["docker-compose.yml"],
  "matchUpdateTypes": ["minor", "major"],
  "enabled": false
}]
```

**Reply by janpio** [reply]
Ah, that of course works. I am always lost when I have to configure things. Thanks for the help as usual!
